test/smoke/testApps/AgentExceptionAndDependency/src/main/java/com/microsoft/ajl/simplecalc/SimpleTrackDependencyAndExceptionServlet.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.ajl.simplecalc;
+
+import javax.servlet.ServletException;
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import redis.clients.jedis.Jedis;
+
+/**
+ * Servlet implementation class SimpleTrackDependencyAndExceptionServlet
```

Comment:
@Terryxxx update the comment here. It should be track RedisDependency and Exception

Reply:
Thanks for your comments. I will change the code.